data/macos.yaml

```diff
@@ -292,6 +313,17 @@ urls:
 - 'http://forensicswiki.org/wiki/Mac_OS_X'
 - 'http://forensicswiki.org/wiki/Mac_OS_X_10.9_-_Artifacts_Location#Kernel_Extension'
 ---
+name: MacOSKnowledgeC
+doc: KnowledgeC User and Application usage database
```

Comment:
KnowledgeC ? what does this stand for?

Maybe add some urls:
https://www.mac4n6.com/blog/2018/8/5/knowledge-is-power-using-the-knowledgecdb-database-on-macos-and-ios-to-determine-precise-user-and-application-usage

Reply:
That DB is known as the "knowledgeC DB" that is why I left it as main name. Given it contains the user application usage, we may change it to either:
1)  "MacOSKnowledgeUserApplicationUsage", although it sounds too long
2) just "MacOSUserApplicationUsage", better although I would have liked to keep the KnowledgeC in the name, But probably this is more useful.

About the URL, yes it is in the URLs note of the mac4n6 artifacts spreadsheet, but I just didn't parse that field in the conversion. I will add it.

Reply:
ok, proper official name is "Duet Knowledge Base", changing the name to reflect this and moved "knowledgeC" and description to "doc:"

Reply:
thx SG